src/main/java/io/specto/hoverfly/junit/dsl/matchers/PlainTextMatcher.java

```diff
@@ -0,0 +1,6 @@
+package io.specto.hoverfly.junit.dsl.matchers;
+
+public interface PlainTextMatcher extends RequestMatcher {
```

Comment:
There's quite a lot of material online about how marker interfaces are an anti-pattern. You could probably achieve this without the marker interface and just use ordinary inheritance.